api-catalog-ui/frontend/src/components/Login/Login.test.jsx

```diff
@@ -75,7 +75,7 @@ describe('>>> Login page component tests', () => {
                 "Authentication problem: 'Username or password are invalid.' for URL '/apicatalog/auth/login'",
             messageKey: 'com.ca.mfaas.security.invalidUsername',
         });
-        expect(messageText).toEqual('Username or password is invalid (SEC0005)');
```

Comment:
Are you sure that the original message was not correct? Does message keys still exist?

Reply:
You're right Jirka, I should actually update them with the new messages defined in the `error-messages.json` file in api-catalog-ui. The message generated is actually the default message when no message is found in the JSON. I will update them, thanks for noticing it!

Reply:
Done! @JirkaAichler